CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ option(ENABLE_COVERAGE "Enable coverage (only supported with GCC)" Off)
 option(ENABLE_FORCEINLINE_RECURSIVE "Enable Forceinline recursive (only supported with Intel compilers)" On)
 option(ENABLE_BENCHMARKS "Build benchmarks" Off)
 option(RAJA_DEPRECATED_TESTS "Test deprecated features" Off)
+option(ENABLE_BOUNDS_CHECK "Enable bounds checking in RAJA::Views/Layouts" Off)
```

Comment:
Can we change this to RAJA_ENABLE_BOUNDS_CHECK for this PR?

Reply:
Sure thing!